### PR TITLE
Make use of jekyll-seo-tag plugin

### PR DESCRIPTION
**Pull request by jhass**
https://github.com/blog/2162-better-discoverability-for-github-pages-sites

### CODE REVIEW

**Comment by asterite** [comment]
Looks good!

@bcardiff merge?

File: _config.yml [hunk 1]
```diff
@@ -1,5 +1,26 @@
-name: Crystal Programming Language
-description: Crystal Programming Language's blog
+name: The Crystal Programming Language
+description: A compiled language with Ruby like syntax and type inference
```
Comment (jhass) [inline _config.yml]: This could use some polishing I guess, open to ideas ;)

Comment (bcardiff) [inline _config.yml]: Yeah, I wasn't sure about the title either. But other than that I <3 it. @asterite?

Comment (asterite) [inline _config.yml]: I guess it's fine, though lately we were trying to avoid the comparison with Ruby...

Comment (jhass) [inline _config.yml]: It doesn't have to be that short either btw, two to three sentences should be fine still.

**Pull request merged**